first-day/intro/index.html

```diff
@@ -182,6 +182,28 @@
 
 <p>4:45-5:30 - Recap on day with buddies and any questions about the Homework Task.</p>
 
+<h3>Week 3:</h3>
+
+<p>9:00-9:30 - Sign in and catch up with buddy - Canteen</p>
+
+<p>9:30-10:30 - Stand-up and recap session from last week (Git, Marshmallow exercise, Wireframing) - Laura + Jameel </p>
+
+<p>10:30-11:30 - Shadow buddy </p>
+
+<p>11:30-12:30 - Team building exercise - All Buddies</p>
+
+<p>12:30-1:30 - Lunch</p>
+
+<p>1:30-2:30 - Planning poker session for website</p>
+
+<p>2.30- 3.00 - Client SDK Stand-up</p>
+
+<p>3.00- 3.15 - Refreshment break</p>
+
+<p>3.15- 4.45 - Javascript session</p>
```

Comment:
Same here?

Reply:
I'm happy to take the javascript session